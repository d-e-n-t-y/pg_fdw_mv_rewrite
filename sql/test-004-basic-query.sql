explain (VERBOSE, COSTS OFF) select * from (select key, COUNT (value) from test group by key) x;
