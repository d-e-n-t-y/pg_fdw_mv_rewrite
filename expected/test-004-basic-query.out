explain (VERBOSE, COSTS OFF) select * from (select key, COUNT (value) from test group by key) x;
                           QUERY PLAN                            
-----------------------------------------------------------------
 Finalize GroupAggregate
   Output: test.key, count(test.value)
   Group Key: test.key
   ->  Sort
         Output: test.key, (PARTIAL count(test.value))
         Sort Key: test.key
         ->  Gather
               Output: test.key, (PARTIAL count(test.value))
               Workers Planned: 2
               ->  Partial HashAggregate
                     Output: test.key, PARTIAL count(test.value)
                     Group Key: test.key
                     ->  Parallel Seq Scan on public.test
                           Output: test.key, test.value
(14 rows)

